rollup-interface/src/state_machine/crypto/simple_merkle.rs

```diff
@@ -0,0 +1,57 @@
+pub const DEFAULT_LEAF_DOMAIN_SEPARATOR: [u8; 1] = [0];
+pub const DEFAULT_INTERNAL_DOMAIN_SEPARATOR: [u8; 1] = [1];
+
+pub trait MerkleHasher {
```

Comment:
Could you please add short description, why this trait is needed?

Reply:
@preston-evans98

Reply:
I haven't yet added a doc comment, but in the meantime I moved it into `fuzzing.rs` so it's less present in the public API.